src/Amalgam/entity/EntityQueryBuilder.h

```diff
@@ -70,11 +82,14 @@ namespace EntityQueryBuilder
 						StringInternPool::StringID feature_type_id = EvaluableNode::ToStringIDIfExists(en);
 						switch(feature_type_id)
 						{
-						case ENBISI_nominal:								feature_type = GeneralizedDistance::FDT_NOMINAL;					break;
-						case ENBISI_continuous:								feature_type = GeneralizedDistance::FDT_CONTINUOUS_NUMERIC;			break;
-						case ENBISI_cyclic:									feature_type = GeneralizedDistance::FDT_CONTINUOUS_NUMERIC_CYCLIC;	break;
-						case GetStringIdFromNodeTypeFromString(ENT_STRING): feature_type = GeneralizedDistance::FDT_CONTINUOUS_STRING;			break;	
-						case ENBISI_code:									feature_type = GeneralizedDistance::FDT_CONTINUOUS_CODE;			break;
+						case ENBISI_nominal_numeric:						feature_type = GeneralizedDistance::FDT_NOMINAL_NUMERIC;			break;
+						case ENBISI_nominal_string:							feature_type = GeneralizedDistance::FDT_NOMINAL_STRING;				break;
+						case ENBISI_nominal_code:							feature_type = GeneralizedDistance::FDT_NOMINAL_CODE;				break;
+						case ENBISI_continuous_numeric:						feature_type = GeneralizedDistance::FDT_CONTINUOUS_NUMERIC;			break;
+						case ENBISI_continuous_numeric_cyclic:				feature_type = GeneralizedDistance::FDT_CONTINUOUS_NUMERIC_CYCLIC;	break;
+						case ENBISI_continuous_string:						feature_type = GeneralizedDistance::FDT_CONTINUOUS_STRING;			break;
+						case ENBISI_continuous_code:						feature_type = GeneralizedDistance::FDT_CONTINUOUS_CODE;			break;
+
```

Comment:
When I stretch this out in on my screen, the tabbing if off between each of these columns (including the breaks).

Reply:
Alright, something really weird happened. This looked all kinds of messed up when I looked on Friday... now it (and the original!) look fine... no idea what that's about. So it was always fine.